virtual-setup/zoom-procedures.md

```diff
@@ -164,9 +182,9 @@ Click `Join Breakout Room`.
 
 If you are assigned to a Zoom breakout room during consultation days, the process for joining a breakout room is the same.
 
-### Sharing your screen in Zoom breakout rooms
+### Sharing your screen in Zoom
 
-Sharing your screen in a Zoom breakout room may be required to allow instructors to best provide assistance.
+Sharing your screen in a Zoom breakout room during instruction may be required to allow instructors to best provide assistance.
```

Comment:
Since the heading got less specific, I thought we could mention the other sharing situations.
```suggestion
Sharing your screen in a Zoom breakout room during instruction or one on one Zoom meetings may be required to allow instructors to best provide assistance.
You will also want to share your screen during your presentation at the end of the workshop.
```
Note also that line 198, "Other participants in the breakout room should now be able to see your screen." should be update to "Other participants in the meeting (or just the breakout room if you are in one) should now be able to see your screen." or similar